packages/editor/src/client/atoms/addBlockInput.tsx

```diff
@@ -34,16 +33,16 @@ export function AddBlockInput({menuItems, subtle, disabled, onMenuItemClick}: Ad
       }}>
       <Dropdown
         disabled={disabled}
-        renderTitle={() => {
-          return <IconButton appearance="primary" icon={<Icon icon="plus" />} circle />
-        }}>
+        renderToggle={(props: object, ref: React.Ref<HTMLButtonElement>) => (
```

Comment:
https://rsuitejs.com/guide/v5-features/#deprecate-code-render-title-code-property-of-code-lt-dropdown-gt-code